core/integration/module_test.go

```diff
@@ -1843,7 +1843,7 @@ class Test:
 		},
 		{
 			sdk: "typescript",
-			source: `import { object, func, field } from "@dagger.io/dagger"
+			source: `import { object, func, func } from "@dagger.io/dagger"
```

Comment:
`func` is imported twice. Happens in several places.

Reply:
Fixed in [58b4d9d](https://github.com/dagger/dagger/pull/7701/commits/58b4d9d46f56661bbf48e4cf03ae5b4353236f6a)